kms-message/src/kms_kmip_result_reason_private.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef KMS_KMIP_RESULT_REASON_PRIVATE_H
+#define KMS_KMIP_RESULT_REASON_PRIVATE_H
+
+#include "kms_message/kms_message_defines.h"
+
+#define KMS_XMACRO                                                                             \
+   KMS_X (ItemNotFound, "Item Not Found", 0x00000001)                                          \
+   KMS_X (ResponseTooLarge, "Response Too Large", 0x00000002)                                  \
+   KMS_X (AuthenticationNotSuccessful, "Authentication Not Successful", 0x00000003)            \
```

Comment:
This file requires ClangFormat. If we want to preserve current format of `KMS_XMACRO`, I recommend adding guards:
```cpp
// clang-format off
#define KMS_XMACRO                                                                             \
   KMS_X (ItemNotFound, "Item Not Found", 0x00000001)                                          \
   /* ... */                                                                                   \
   KMS_X (ObjectAlreadyExists, "Object Already Exists", 0x00000018)                            \
   KMS_X_LAST (GeneralFailure, "General Failure", 0x00000100)
// clang-format on
```